src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -88,12 +93,20 @@ const Body = BaseForm.extend({
       publicKey: options,
       signal: this.webauthnAbortController.signal
     }).then((assertion) => {
+      const credentials = {
+        clientData: CryptoUtil.binToStr(assertion.response.clientDataJSON),
+        authenticatorData: CryptoUtil.binToStr(assertion.response.authenticatorData),
+        signatureData: CryptoUtil.binToStr(assertion.response.signature)
+      };
+      const hasUserHandleSchema = this.options.appState.getSchemaByName('credentials.userHandle');
+      if (hasUserHandleSchema) {
+        _.extend(credentials, {
+          userHandle: CryptoUtil.binToStr(assertion.response.userHandle ?? '')
```

Comment:
out of curiosity. what does it mean this line?  specially the double 
`??` `CryptoUtil.binToStr(assertion.response.userHandle ?? '')`.

Reply:
according to webauthn spec, userHandle is nullable, so I use this `nullish coalescing operator` to handle null case

Reply:
awesome, I just learned a new operator in JS, thank you @xifengfang-okta